rpcserver.go

```diff
@@ -799,6 +799,23 @@ func (r *rpcServer) ListUnspent(ctx context.Context,
 	return resp, nil
 }
 
+// activeNetParams returns a string representation of the currect active
+// network.
+func activeNetName() (string, error) {
+	switch {
+	case cfg.Bitcoin.MainNet || cfg.Litecoin.MainNet:
+		return "mainnet", nil
+	case cfg.Bitcoin.TestNet3 || cfg.Litecoin.TestNet3:
+		return "testnet", nil
+	case cfg.Bitcoin.SimNet || cfg.Litecoin.SimNet:
+		return "simnet", nil
+	case cfg.Bitcoin.RegTest || cfg.Litecoin.RegTest:
+		return "regest", nil
```

Comment:
was casually browsing for issues on lnd and noticed the typo here. `regest`

Reply:
thanks for the catch! fixed.